src/common/SearchResults/index.js

```diff
@@ -0,0 +1,14 @@
+import { useLocation } from "react-router-dom";
+import { Section, SectionTitle } from "../Section";
+
+export const SearchResults = () => {
+	const location = useLocation();
+	const searchParams = new URLSearchParams(location.search);
+	const query = searchParams.get("query") || "";
```

Comment:
Kiedyś można by stworzyć taką funkcję, która przyjmuje w argumencie na przykład query lub page i zwraca automatycznie to co jest w URL, ale nie wiem jak często będziemy pobierać jakieś parametry z URL, więc narazie chyba nie ma sensu

export const useURLParameter = (argument) => {
    const location = useLocation();

    return (new URLSearchParams(location.search)).get(argument);
};